### ISSUE TITLE
Add "key results" to model

### ISSUE THREAD

**Issue by denised** [ProjectDrawdown/solutions#304]
The original spreadsheets have a "key results" section on the top of the Advanced Controls tab, which is a quick way to see what a scenario is producing.  These calculations are not done in the model code (though in most cases they are a fairly simple summation).  Many of them _are_ done in the Jupyter Notebook in the limbo directory (the code for which can be found in the limbo/ui/charts.py).

It would be nice to have these results easily available from a Scenario object, as they provide a quick and easy-to-understand overview of the results.  They would also serve as a quick way to compare output between the python models and the spreadsheet models.   Perhaps the scenario object could return a dictionary.  

The most important results are illustrated in Advanced Controls!A2:F6, but note that there is an extended set from A7:G21 that would also be good to have.

**Comment by denised** [comment]
As an example, here are the functions I copied from the UI code to evaluate a few of the results.  As you can see the functions are not very complex, but just complicated enough that a newcomer to the code would have a hard time figuring out how to do them:
```python
def net_operating_savings(s):
    return (
        (s.oc.conv_ref_cumulative_operating_cost().loc[ 2050 ] -
         s.oc.conv_ref_cumulative_operating_cost().loc[ 2020 ]) -
        (s.oc.soln_pds_cumulative_operating_cost().loc[ 2050 ] -
         s.oc.soln_pds_cumulative_operating_cost().loc[ 2020 ])   
        ) / 10**9

def cumulative_net_first_cost(s):
    return (
        s.fc.soln_pds_annual_world_first_cost().loc[ :2050].sum() -
        s.fc.soln_ref_annual_world_first_cost().loc[ :2050].sum() -
        s.fc.conv_ref_annual_world_first_cost().loc[ :2050].sum()
    ) / 10**9

def cumulative_emissions_reduced(s):
    return s.c2.co2eq_mmt_reduced().loc[2020:2050, 'World'].sum() / 1000
```

**Comment by denised** [comment]
Officially it is a "Nice to have", but I think we will end up doing it because it will help with our third goal of making it possible for researchers to use the code.

-- 
  Denise Draper
  ***@***.***

**Comment by denised** [comment]
Closed as duplicate of #426
